src/mds/CInode.h

```diff
@@ -394,11 +394,12 @@ class CInode : public MDSCacheObject, public InodeStoreBase, public Counter<CIno
   static const int MASK_STATE_REPLICATED = STATE_RANDEPHEMERALPIN;
 
   // -- waiters --
-  static const uint64_t WAIT_DIR         = (1<<0);
-  static const uint64_t WAIT_FROZEN      = (1<<1);
-  static const uint64_t WAIT_TRUNC       = (1<<2);
-  static const uint64_t WAIT_FLOCK       = (1<<3);
-  
+  static const uint64_t WAIT_DIR           = (1<<0);
+  static const uint64_t WAIT_FROZEN        = (1<<1);
+  static const uint64_t WAIT_TRUNC         = (1<<2);
+  static const uint64_t WAIT_FLOCK         = (1<<3);
+  static const uint64_t WAIT_UNLINK_FINISH = (1<<4);
```

Comment:
s/WAIT_UNLINK_FINISH/WAIT_UNLINK/ ? To look similar to others ?

Reply:
Looks good. Done. Thanks!